service/frontend/workflow_handler.go

```diff
@@ -467,6 +467,99 @@ func (wh *WorkflowHandler) unaliasedSearchAttributesFrom(
 	return sa, nil
 }
 
+func (wh *WorkflowHandler) ExecuteMultiOperation(
+	ctx context.Context,
+	request *workflowservice.ExecuteMultiOperationRequest,
+) (_ *workflowservice.ExecuteMultiOperationResponse, retError error) {
+	defer log.CapturePanic(wh.logger, &retError)
+
+	if request == nil {
+		return nil, errRequestNotSet
+	}
+
+	if !wh.config.EnableExecuteMultiOperation(request.Namespace) {
+		return nil, errMultiOperationAPINotAllowed
+	}
+
+	// TODO: validation
+
+	namespaceName := namespace.Name(request.GetNamespace())
```

Comment:
nit: you can use `request.Namespace` directly here as you've already ensured request isn't nil. There's no reason to call these getters